db/models.py

```diff
@@ -14,3 +14,44 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movies_actor")
+    genres = models.ManyToManyField(Genre, related_name="movies_genre")
+
+    def __str__(self) -> str:
+        return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self) -> str:
+        return self.name
+
+    @property
+    def capacity(self) -> int:
+        return self.rows * self.seats_in_row
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField()
+
+    cinema_hall = models.ForeignKey(
+        CinemaHall,
+        on_delete=models.CASCADE,
+        related_name="movie_sessions_halls"
```

Comment:
The `related_name` for the `cinema_hall` field in `MovieSession` is set to 'movie_sessions_halls'. This might be slightly misleading as it suggests multiple halls. Consider using a singular form like 'movie_sessions' to better reflect the relationship.